build.ps1

```diff
@@ -85,10 +85,6 @@ function MD5HashFile([string] $filePath)
 
 Write-Host "Preparing to run build script..."
 
-if(!$PSScriptRoot){
-    $PSScriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent
-}
```

Comment:
This and the usage on line 45 would break it for PowerShell 2 users, but I doubt there are any users trying to build this on Windows 7 without service pack 🙂